railties/lib/rails/generators/rails/app/templates/config/initializers/new_framework_defaults_6_0.rb.tt

```diff
@@ -8,3 +8,10 @@
 
 # Don't force requests from old versions of IE to be UTF-8 encoded
 # Rails.application.config.action_view.default_enforce_utf8 = false
+#
+# Purpose and expiry information are embedded inside signed and encrypted
```

Comment:
Let's make the sentence more active (and why not swap information with metadata too): `Embed purpose and expiry metadata inside signed and…`